static/js/twitter-filter.js

```diff
@@ -0,0 +1,30 @@
+function twitter_filter() {
+  var twitter_handles = []
+  for (var i = 0; i < document.getElementsByClassName('org-info').length; i++) {

```

Comment:
You are using `document.getElementsByClassName('org-info')` more than one times. Don't Repeat Yourself (DRY). Makes it more readable by creating a variable like `orgs`.